src/collective/nitf/upgrades/v1008/__init__.py

```diff
@@ -12,7 +12,10 @@ def fix_collections(context):
     logger.info('About to update {0} objects'.format(len(collections)))
     for col in collections:
         obj = col.getObject()
-        query = obj.getQuery()
+        try:
+            query = obj.getRawQuery()
```

Comment:
@rodfersou IMO, this has to be fixed in another way; the original error was:

```python
Traceback (innermost last):
  Module ZPublisher.Publish, line 138, in publish
  Module ZPublisher.mapply, line 77, in mapply
  Module ZPublisher.Publish, line 48, in call_object
  Module Products.CMFCore.FSPythonScript, line 127, in __call__
  Module Shared.DC.Scripts.Bindings, line 322, in __call__
  Module Shared.DC.Scripts.Bindings, line 359, in _bindAndExec
  Module Products.PythonScripts.PythonScript, line 344, in _exec
  Module script, line 11, in prefs_reinstallProducts
   - <FSPythonScript at /Plone/prefs_reinstallProducts used for /Plone/portal_quickinstaller>
   - Line 11
  Module Products.CMFPlone.QuickInstallerTool, line 101, in upgradeProduct
  Module Products.GenericSetup.upgrade, line 140, in doStep
  Module collective.nitf.upgrades.v1008, line 18, in fix_collections
TypeError: 'CatalogContentListingObject' object has no attribute '__getitem__'
```

so I don't understand why are you using an `AttributeError` exception.

we should call `obj.getQuery()` and catch the `TypeError` exception instead.

also, the final fix must be ported to master branch.